src/App.tsx

```diff
@@ -1,42 +1,34 @@
-import React from 'react';
+import { FC, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
+import { Tab } from './types';
+import { Tabs } from './components/Tabs';
+import { tabs } from './tabs';
 
-export const tabs = [
-  { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
-  { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
-  { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
-];
+export const App: FC = () => {
+  const [selectedTabId, setSelectedTabId] = useState(tabs[0].id);
+
+  const onTabSelected = (tab: Tab) => {
+    if (tab.id === selectedTabId) {
+      return;
+    }
+
+    setSelectedTabId(tab.id);
+  };
+
+  const selectedTab = tabs.find(tab => tab.id === selectedTabId);
```

Comment:
If you get the whole object here, pass it to the child component, then you won't need to pass the ID.

I mean, you are looking for selectedTab in tabs twice, once here and second time in Tab component. You don't need to, you can do it just here, and pass it as a prop

Reply:
This is actually what I intended to do in the beginning of solving this task, but unfortunately the task description says:

> 1. Save the `selectedTabId` in the `App` (the first `tab` is selected by default);
and
> 1. Pass the `selectedTabId` as a prop to the `Tabs`

I think it may be mandatory to do it this way for the tests to go through, because when I am trying to pass the whole selectedTab object as a props, instead of just suing selectedTabId the tests would not pass.

Reply:
The same, because at this moment we do not yet have a changed state, so we have to do this check and calculate the current table again 

instead of passing the whole tab

Reply:
I found a workaround by moving all the JSX from the App component to the Tab component. It might not be the most elegant solution, but it functions properly; the tests pass. Now there's no need to pass the entire object as a prop or to check for the selectedTab in the App component.